libs/ballot-interpreter-nh/ts/src/cli.ts

```diff
@@ -272,10 +276,10 @@ function tryReadElectionFromElectionTable(
       db
         .prepare('SELECT election_data as electionData FROM election LIMIT 1')
         .get() as Optional<{ electionData: string }>
-    )?.electionData;
+    )?.electionData?.toString();
 
     return electionData
-      ? safeParseElectionDefinition(electionData).ok()
+      ? { electionData, election: JSON.parse(electionData), electionHash: '' }
```

Comment:
Why this change?

Reply:
This was part of an unrelated change that I had mostly backed out. I forgot about this piece, but I'll revert. The reason for it was because I was trying to work with a backup that had an incompatible version of the `Election` struct.